lxd/storage_volumes.go

```diff
@@ -623,6 +623,39 @@ func storagePoolVolumesTypePost(d *Daemon, r *http.Request) response.Response {
 		return response.Conflict(fmt.Errorf("Volume by that name already exists"))
 	}
 
+	target := queryParam(r, "target")
+
+	// Check if we need to redirect to migration
```

Comment:
Please dont use the word "redirect" in an API handler unless you mean HTTP redirection. 
This caught me by surprise as there is no HTTP redirect happening here.

Reply:
Maybe try "switch to" instead.